index.tt

```diff
@@ -92,7 +92,7 @@
       The Nix Packages collection
       <strong>(<a href="https://github.com/NixOS/nixpkgs">Nixpkgs</a>)</strong>
       is a set of
-      <strong class="-highlighted">over 60&#160;000 packages</strong> for the Nix package manager.
+      <strong class="-highlighted">over 80&#160;000 packages</strong> for the Nix package manager.
```

Comment:
Not sure why `&nbsp;` was used here.

```suggestion
      <strong class="-highlighted">over 80,000 packages</strong> for the Nix package manager.
```

Reply:
I just saw this commit 86b2e28c9aace45485dacdc7e302870bb1254c64

Reply:
Looks good, thanks

Reply:
@tuxiqae Nope, sorry, this broke the build. For some reason something in there doesn't accept `&nbsp;`. Would be great if you could force-push your original version over this, then I'll merge it.

Reply:
Sure, but I will only be able to do so in a couple of hours.

Reply:
The reason is that `&nbsp;` is HTML, and this is handled through an XML-based tooling.